src/reducers/subCategorizeAll.ts

```diff
@@ -4,6 +4,16 @@ import { State } from '../util/initialState'
 import { parentOf, ellipsize, headValue, isEM, pathToContext, once, reducerFlow, isRoot } from '../util'
 import { getChildrenRanked, hasChild, lastThoughtsFromContextChain, simplifyPath, splitChain } from '../selectors'
 
+const SUBCATEGORIZE_ALL_META_EXCEPTIONS = [
+  '=archive',
+  '=bullet',
+  '=focus',
+  '=label',
+  '=note',
+  '=pin',
+  '=publish',
+]
```

Comment:
Can you use an object for this? The array is O(n) lookup while an object is O(1).

```ts
const SUBCATEGORIZE_ALL_META_EXCEPTIONS = {
  '=archive': true,
  '=bullet': true,
  ...
}
```

```ts
const filteredChildren =  childen.filter(child => child.value in SUBCATEGORIZE_ALL_META_EXCEPTIONS)
```